setup.php

```diff
@@ -1,203 +1,239 @@
 <?php
 
-function plugin_quicktree_version()
-{
-    global $config;
-    $info = parse_ini_file($config['base_path'] . '/plugins/quicktree/INFO', true);
-    return $info['info'];
+function plugin_quicktree_install() {
+	api_plugin_register_hook('quicktree', 'top_header_tabs',          'quicktree_show_tab',             'setup.php');
+	api_plugin_register_hook('quicktree', 'top_graph_header_tabs',    'quicktree_show_tab',             'setup.php');
+	api_plugin_register_hook('quicktree', 'config_arrays',            'quicktree_config_arrays',        'setup.php');
+	api_plugin_register_hook('quicktree', 'config_settings',          'quicktree_config_settings',      'setup.php');
+	api_plugin_register_hook('quicktree', 'draw_navigation_text',     'quicktree_draw_navigation_text', 'setup.php');
+	api_plugin_register_hook('quicktree', 'graph_buttons',            'quicktree_graph_buttons',        'setup.php');
+	api_plugin_register_hook('quicktree', 'graph_buttons_thumbnails', 'quicktree_graph_buttons',        'setup.php');
+
+	api_plugin_register_hook('quicktree', 'page_head', 'quicktree_page_head', 'setup.php');
+
+	quicktree_setup_table();
+
+	return true;
 }
```

Comment:
Not sure why this was moved to the start of the file, but it causes some differencing issues and makes it harder to see if any change actually occurred.

Reply:
Just cause that's the way it is on just about every plugin.